crepecake/src/main/java/com/google/cloud/tools/crepecake/cache/CacheWriter.java

```diff
@@ -70,4 +79,52 @@ public CachedLayer writeLayer(UnwrittenLayer layer) throws IOException {
       return new CachedLayer(layerFile, compressedBlobDescriptor, diffId);
     }
   }
+
+  /**
+   * Writes a {@link ReferenceLayer} with its corresponding compressed layer content BLOB to cache.
+   */
+  public CachedLayer writeLayer(ReferenceLayer layer, Blob layerContent) throws IOException {
+    File layerFile = getLayerFile(layer.getBlobDescriptor().getDigest());
+
+    // Writes the layer content to file.
+    try (OutputStream fileOutputStream =
+        new BufferedOutputStream(new FileOutputStream(layerFile))) {
+      layerContent.writeTo(fileOutputStream);
+    }
+    // TODO: Should probably check if the written BLOB has the same digest as expected.
```

Comment:
Why? Are you concerned it may write the wrong bytes? Is this a plausible scenario?

If so, implement now or file a bug. Otheerwise, drop the TODO.

Reply:
This is mostly just to make sure that the parameters are valid - that the `layerContent` passed in was really the BLOB for the `layer`. 

FIled https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/issues/102 to address this.